app/models/affiliate.rb

```diff
@@ -0,0 +1,8 @@
+class Affiliate < ApplicationRecord
+
+    has_many :coordinators
+    has_many: tutors
+
+    has_many :matches
```

Comment:
Curious why you went this route.
If we have an unmatched student, wouldn't this mean that the student didn't have any way to connect to the affiliate?